packages/sui-ssr/build-device.js

```diff
@@ -0,0 +1,8 @@
+export const buildDeviceFrom = ({request = {}, window = {}}) => {
+  return {
+    // https://tc39.github.io/proposal-optional-chaining/
+    'user-agent':
```

Comment:
Why not returning it camelCase?

Reply:
No reason, just because is the header´s name

Reply:
If is not a huge change or painful I'll change it by camelCase in order to be more compliant with the var names standards